### PR TITLE
adds kathryn gronsbell to mediaconch blog

### PR DESCRIPTION
**Pull request by ablwr**
Please review and deploy! Note the three images included. Image file paths should be good to go. Content approved by @dericed and @JeromeMartinez but cursory glance for Markdown errors is always appreciated.

### CODE REVIEW

**Review by dericed — APPROVED** [review]
The image relative links start with a slash, which seems odd but maybe it's fine. As long as the image embeds work, LGTM.

File: _posts/2017-07-07-interview-with-kathryn-gronsbell.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title:  "Interview with Kathryn Gronsbell"
+date:   2017-06-13
```
Comment (dericed) [inline _posts/2017-07-07-interview-with-kathryn-gronsbell.md]: bump the date?

Comment (ablwr) [reply _posts/2017-07-07-interview-with-kathryn-gronsbell.md]: Good catch, thank you, my inability to remember this part ruins my life regularly.

**Comment by ablwr** [comment]
FWIW preview will work locally with --baseurl='/MediaConch' but not if you try to peep on a gh-pages site for this PR. I think that relative path is what we want considering this gets mirrored on a different server. It's what we do with the other functioning images. LMK if we should change, those in charge of the mysterious deployment process.

**Comment by GuillaumeRoques** [comment]
It's deployed on prod, images path are good, thanks.

**Comment by ablwr** [comment]
Thanks @GuillaumeRoques!!!!!!!

**Pull request merged**